app/src/main/java/com/example/pocketcloset/MainActivity.java

```diff
@@ -34,9 +42,33 @@ public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        try {
+            PackageInfo info = getPackageManager().getPackageInfo(
+                    "com.example.pocketcloset",
+                    PackageManager.GET_SIGNATURES);
+            for (Signature signature : info.signatures) {
+                MessageDigest md = MessageDigest.getInstance("SHA");
+                md.update(signature.toByteArray());
+                Log.d("KeyHash:", Base64.encodeToString(md.digest(), Base64.DEFAULT));
+            }
+        }
+        catch (PackageManager.NameNotFoundException e) {
+        }
+        catch (NoSuchAlgorithmException e) {
+        }
```

Comment:
What does this do?

Reply:
This allowed me to get the key hash for the facebook sdk, but I can remove it now!